### PR TITLE
Forces calculation for compressible, multiphase and turbulence

### PR DESCRIPTION
**Pull request by JSeuffert**
Concerning issues #57 #42 
Maybe in parts the same as pull request #51 

Forces calculation is implemented as in the forces functionObject.

I tested the correct implementation with: pimpleDyMFoam, rhoPimpleDyMFoam, interDyMFoam, compressibleInterDyMFoam; laminar flow and RAS turbulence model
Tested with Openfoam 4 (https://github.com/OpenFOAM/OpenFOAM-4.x), precice 1.3.0, Calculix 2.13

Forces calculation itself is not verified!

### CODE REVIEW

**Comment by derekrisseeuw** [comment]
Incredible!! 
The verification of the incompressible variant of the forces is done in my thesis, which will be available shortly.

**Comment by MakisH** [comment]
Thank you very much, @JSeuffert! Is this ready to review? I will try to have a look next week.

**Comment by JSeuffert** [comment]
I just deleted my last two commits that should not be a part of this pull request. 
It's now ready to review!

File: FSI/Force.H [hunk 1]
```diff
@@ -5,6 +5,13 @@
 
 #include "fvCFD.H"
 
+// TEMPORARY
```
Comment (MakisH) [inline FSI/Force.H]: Maybe check?

**Pull request merged**